### PR TITLE
Adding Customer Risk Section

### PR DESCRIPTION
**Pull request by emilyhollinger**

### CODE REVIEW

**Comment by justinegeffen** [comment]
Thanks, @emilyhollinger! I edited the doc directly with punctuation and spacing fixes, and wanted to check on any trailing spaces. One suggestion, otherwise LGTM!

File: operations/customer-success/README.md [hunk 1]
```diff
@@ -129,3 +132,31 @@ Below are detailed steps on how to log a Customer Reference in Salesforce.
    - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that the mattermost cannot use their logo or publish case studies.
```
Comment (justinegeffen) [inline operations/customer-success/README.md]:
```suggestion
   - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that Mattermost cannot use their logo or publish case studies.
```

File: operations/customer-success/README.md [hunk 2]
```diff
@@ -129,3 +132,31 @@ Below are detailed steps on how to log a Customer Reference in Salesforce.
...
    - **Notes:** Field to add any additional details.
 4. Select **Save**.
+
+### Customer Risk
+
+Risk is the chance or probability that a customer will non-renew their existing Enterprise license. The Customer Success team categorizes risk into two buckets:
```
Comment (justinegeffen) [inline operations/customer-success/README.md]: I understand "non-renew" is the term we use, but wondering if we could say "not renew" in this case? 0/5.

**Review by justinegeffen — DISMISSED** [review]

File: operations/customer-success/README.md (hunk 2 above)
Comment (emilyhollinger) [reply operations/customer-success/README.md]: @justinegeffen I'm good with that change.

File: operations/customer-success/README.md (hunk 1 above)
Comment (emilyhollinger) [reply operations/customer-success/README.md]: @justinegeffen I'm good with this change.

**Comment by emilyhollinger** [comment]
@justinegeffen I'm good with all changes.

File: operations/customer-success/README.md [hunk 3]
```diff
@@ -126,6 +129,34 @@ Below are detailed steps on how to log a Customer Reference in Salesforce.
...
    - **Owner:** Automatically populated and assigned to the person who creates the reference. It can be changed by using the **Change Owner** button.
    - **Approved by Customer Legal:** Customer contact has approval from their legal team.
-   - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that the mattermost cannot use their logo or publish case studies.
+   - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that Mattermost cannot use their logo or publish case studies.
    - **Notes:** Field to add any additional details.
 4. Select **Save**.
+
+### Customer Risk
+
+Risk is the chance or probability that a customer will non-renew their existing Enterprise license. The Customer Success team categorizes risk into two buckets:
```
Comment (justinegeffen) [inline operations/customer-success/README.md]:
```suggestion
Risk is the chance or probability that a customer will not renew their existing Enterprise license. The Customer Success team categorizes risk into two buckets:
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, Emily! LGTM. :)

**Review by rbradleyhaas — APPROVED** [review]

**Pull request merged**